docs/_layouts/default.html

```diff
@@ -3,15 +3,22 @@
   <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="viewport" content="width=device-width">
```

Comment:
Why is this on line 6 and 16? Is this line needed? It doesn't seem to do anything if it's deleted. But happy to learn something.